### PR TITLE
381 edit tag change image when song is changed

### LINKED ISSUE #381: Edit Tag: Change Image when Song is changed

### PR DESCRIPTION
**Pull request by schnickalodeon**
closes #381 

Change the image if track is changed **and** the selected image isn't an external one

### CODE REVIEW

**Review by tcoro — APPROVED** [review]
n 🍨 🧊 🍦

**Review by anbraten — CHANGES_REQUESTED** [review]
Nice change.
One bug I noticed. If you select a tag at the home screen and choose edit your are getting redirect to the track selection instead of the tag-details page.

And one question: If I selected a special custom image I really like and now switch the track I would loose my image as well?

File: packages/app/src/main.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import '@fortawesome/fontawesome-free/css/all.css';
 import '../index.css';
-import '@fontsource/roboto/700.css'; // TODO disable font to get rid of an error
+import '@fontsource/roboto/400.css'; // TODO disable font to get rid of an error
 import '@fontsource/baloo-2/700.css'; // TODO disable font to get rid of an error
```
[the remark this answers is not in the document]
Comment (schnickalodeon) [reply packages/app/src/main.ts:4]: Thanks therefore. Strangely enough, I've encoutered that bug as well and changed that. But that wasn't merged ^^

And No, the image will only be replaced if the former image was an Spotify-Image (cp. Description ;)

**Review by anbraten — CHANGES_REQUESTED** [review]
🤔  this change doesn't work for me.

**Comment by schnickalodeon** [comment]
> 🤔 this change doesn't work for me.

Yes I see. I found the issue

**Review by JanEberlein — APPROVED** [review]
Looks good 👍

**Review by anbraten — APPROVED** [review]

**Pull request merged**